java/streaming/src/main/java/com/ray/streaming/core/runtime/collector/RayCollector.java

```diff
@@ -0,0 +1,54 @@
+package com.ray.streaming.core.runtime.collector;
+
+import com.ray.streaming.api.collector.Collector;
+import com.ray.streaming.api.partition.IPartition;
+import com.ray.streaming.core.graph.ExecutionEdge;
+import com.ray.streaming.core.graph.ExecutionGraph;
+import com.ray.streaming.core.runtime.StreamWorker;
+import com.ray.streaming.message.Message;
+import com.ray.streaming.message.Record;
+import java.util.Arrays;
+import java.util.Map;
+import org.ray.api.Ray;
+import org.ray.api.RayActor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class RayCollector implements Collector<Record> {
```

Comment:
Please add some documentation.

Reply:
ok, i will add some documentation

Reply:
@raulchen yes, if we use Ray Pipe to send message, we can implement RayPipeCollector